### DISCUSSION TITLE
Default install location

### DISCUSSION THREAD

**Discussion by boomshankerx** [JanDeDobbeleer/oh-my-posh#5738] [Troubleshoot] [answered]
After reading the documentation I was expecting the install script to attempt to install to /usr/local/bin on a linux system however it appears that the docs are out of date. The default installation location now seems to be ~/.local/bin. Can someone confirm if the documentation should be updated.

**Comment by JanDeDobbeleer** [comment]
@boomshankerx yes, the docs should he updated indeed.

**Reply by JanDeDobbeleer** [reply, marked as answer]
@boomshankerx docs are updated.
